src/directives/view.js

```diff
@@ -257,8 +259,27 @@ class routeViewDirective {
         if (component.controller) {
           const locals = _.merge(dependencies, {$scope: viewScope, $element: element.children().eq(0)});
 
-          const controller = $controller(component.controller, locals);
-          locals.$scope[component.controllerAs] = controller;
+          try {
```

Comment:
Error handling so that we actually see errors when the constructor fails to be instantiated... not specifically related to this change but we agreed to do it the next time someone touched the router.